swift_package/BrainFlow/Tests/BrainFlowTests/BrainFlowTests.swift

```diff
@@ -0,0 +1,11 @@
+import XCTest
+@testable import BrainFlow
+
+final class BrainFlowTests: XCTestCase {
+    func testExample() throws {
+        // This is an example of a functional test case.
```

Comment:
this file looks like a copypaste from examples, probably should not be in the repo

Reply:
removed